DHIS2/cloner/dhis2_clone

```diff
@@ -130,6 +138,19 @@ def magenta(txt):
     return '\x1b[35m%s\x1b[0m' % txt
 
 
+def execute_scripts(scripts_folder):
+    scripts = []
+    files = os.listdir(scripts_folder)
+    for file in files:
+        filename, file_extension = os.path.splitext(file)
+        if file_extension in ('.sh','.py'):
```

Comment:
In the same vein, you could say:

```
_, ext = os.path.splitext(fname)
if ext in ['.sh', '.py']:
```

to make explicit you are disregarding the first output, and also to use lists for homogeneous data (a very very minor comment, just to be more pythonic).

Reply:
thanks for that. Agreed